docs/index.rst

```diff
@@ -3,6 +3,18 @@
 Installing using Docker
 =======================
 
+.. toctree::
+    :maxdepth: 2
+    :hidden:
+
+    quickstart
+    security/index
+    configuration
+    operations/index
```

Comment:
it seems like this repo wasn't following the convention of using `index.rst`, so these probably all need to be modified to use the corresponding filename within each directory.